R/data.R

```diff
@@ -97,6 +97,14 @@ NULL
 #' @format NULL
 "growth_rates_floodplain"
 
+#' Prey Density
```

Comment:
Can I merge the prey density PR before review so that we do do not see the changes that I already reviewed in that one?